src/frontend/components/PublicPausedLiveVideo/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Clock, Layer, Paragraph } from 'grommet';
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import videojs from 'video.js';
+
+import { H2 } from '../Headings';
+import { Video } from '../../types/tracks';
+import { useAsyncEffect } from 'utils/useAsyncEffect';
+import { resumeLive } from '../../utils/resumeLive';
+
+const messages = defineMessages({
+  pausedSince: {
+    defaultMessage: 'The webinar is in paused since ',
+    description: 'Information about how many time the webinar is paused.',
```

Comment:
not sure but I tried to lookup the way to say that and would conclude:
```suggestion
    description: 'Information about the time since which the webinar is paused.',
```